en_US/faq/faq.md

```diff
@@ -76,34 +76,34 @@ Hook is an interface provided by EMQX, which will be triggered when a connection
 
 
 
-## What's mqueue? How to use mqueue in EMQX?
+## What's an mqueue? How to configure mqueues in EMQX?
```

Comment:
```suggestion
## What's a mqueue? How to configure mqueues in EMQX?
```

Reply:
Does this depend on how mqueue is pronounced (emqueue or mqueue)?

Reply:
Good question :thinking: